.circleci/config.yml

```diff
@@ -81,6 +81,42 @@ jobs:
           paths:
             - ./vendor
 
+  release:
+    working_directory: ~/.go_workspace/src/github.com/maistra/istio-workspace
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - run:
+          name: "Sets up Quay.io"
+          command: |
+            mkdir -p ~/.docker && chmod 0700 ~/.docker
+            touch ~/.docker/config.json && chmod 0600 ~/.docker/config.json
+            echo $QUAY_AUTH_JSON > ~/.docker/config.json
```

Comment:
As long as we have this option turned off we are good with this approach (and in fact we do have it disabled! \o/)
![image](https://user-images.githubusercontent.com/719616/60463945-92c04600-9c4d-11e9-9d3e-54e3f671677c.png)